### PR TITLE
#5130 Uint16

### PR DESCRIPTION
**Pull request by trihernandez**
- [ ] #5130

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/name_space_functions.py [hunk 1]
```diff
@@ -115,4 +117,4 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False, *, where=Non
     ret = ivy.var(a, axis=axis, correction=ddof, keepdims=keepdims, out=out)
     if ivy.is_array(where):
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
-    return ret.astype(dtype)
+    return ret.astype(dtype)
```
Comment (zaeemansari70) [inline ivy/functional/frontends/jax/numpy/name_space_functions.py:120]: There should be a new line here.

File: ivy_tests/conftest.py [hunk 2]
```diff
@@ -39,4 +39,4 @@ def pytest_collection_modifyitems(items):
         for id_ in skip_ids:
             if item.nodeid.startswith(id_):
                 item.add_marker(skip_ivy)
-                break
+                break
```
Comment (zaeemansari70) [inline ivy_tests/conftest.py:42]: Here too, No change should be there in the `conftest.py` file.

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hey! 
First of all thank you soo much for contributing to ivy! 🙂
Your PR looks amazing ✨!
Just requested a few changes and left comments on them which needs to be addressed before we can merge your PR! 🙂

1. It is giving an error for torch `Cannot convert to PyTorch dtype. uint16 is not supported by PyTorch.` 
2. There are a bunch of lint errors I've pointed out, they need to be fixed as well.
3. You've also committed a change in the conftest file please uncommit that change too.

Thanks!🙂

**Comment by trihernandez** [comment]
I did the last two.  What would be the best way to interpret uint16 in pytorch (such as using an int32 datatype or two uint8 datatypes), or convert a value in pytorch to uint16 (even if this new data value is not using a pytorch datatype)?  Would I take the literal byte data of the uint16 and express that as the byte data for a signed 16-bit int?

**Comment by trihernandez** [comment]
I did the last two. What would be the best way to interpret uint16 in pytorch (such as using an int32 datatype or two uint8 datatypes), or convert a value in pytorch to uint16 (even if this new data value is not using a pytorch datatype)? Would I take the literal byte data of the uint16 and express that as the byte data for a signed 16-bit int? How would I add a separate case for pytorch compared to the other data types?

When I ran the test cases on codespaces, my part of the code passed, so I do not know why my code is failing the test with this error now, especially since I worked on the jax frontend.

I read something that might be relevant to excluding pytorch here (https://lets-unify.ai/ivy/deep_dive/6_data_types.html#supported-and-unsupported-data-types), but it seems to be backend-related to it.

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Just one more small change before we can merge your PR 😊 

1. Please un-commit the `ivy_array.pickle` file.

Also, no need to worry about the dtype issue for Torch, it will surely be handled by the `Ivy` team in the future.
Thanks! 😊

**Comment by trihernandez** [comment]
Deleted the pickle file.  You should be good to commit!

**Review by zaeemansari70 — APPROVED** [review]
Lgtm!

**Pull request merged**